src/lib/php/Dao/CopyrightDao.php

```diff
@@ -163,19 +163,26 @@ public function undoDecision($tableName,$pfileId, $decisionId)
   public function getScannerEntries($tableName, $uploadTreeTableName, $uploadId, $type, $extrawhere)
   {
     $statementName = __METHOD__.$tableName.$uploadTreeTableName;
-    $params[]= $uploadId;
+    $params = array();
+    $extendWClause = null;
 
-    $whereClause = null;
-    if ($type !== null && $type != "skipcontent")
+    if ($uploadTreeTableName === "uploadtree_a")
+    {
+      $params[]= $uploadId;
+      $extendWClause .= " AND UT.upload_fk = $".count($params);
+      $statementName .= ".withUI";
+    }
+
+    if($type !== null && $type != "skipcontent")
     {
       $params[]= $type;
-      $whereClause .= " AND C.type = $".count($params);
+      $extendWClause .= " AND C.type = $".count($params);
       $statementName .= ".withType";
     }
 
     if ($extrawhere !== null)
     {
-      $whereClause .= "AND ". $extrawhere;
+      $extendWClause .= "AND ". $extrawhere;
```

Comment:
Please add a space before the `AND` as it may cause issue during concatenation.

Reply:
Space added